spec/features/galleries/show_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe GalleriesController, type: :feature do
```

Comment:
Good feature testing. Though maybe for the future could use a let! instead of before :each.